ospec/ospec.js

```diff
@@ -49,6 +49,9 @@ module.exports = new function init(name) {
 		spy.callCount = 0
 		return spy
 	}
+	o.cleanStackTrace = function(stack) {
+		return stack.match(/^(?:(?!Error|[\/\\]ospec[\/\\]ospec\.js).)*$/gm).pop()
```

Comment:
Not a critical nit, but this isn't safe for if `ospec` is bundled. (This isn't a problem for Mithril, but some of its users might get tripped up on it.)

Reply:
I... don't really get it. I just placed that funtion there cause I wanted to have it accessible in the test, but still encapsulated within the o object. I'd happily move it.

Reply:
To clarify, the code snippet I'm referring to was there originally.